synapse_invite_checker/invite_checker.py

```diff
@@ -269,6 +269,16 @@ def parse_config(config: Dict[str, Any]) -> InviteCheckerConfig:
                 msg = "`tim-type` setting is not a recognized value. Please fix."
                 raise ConfigError(msg)
 
+        _allowed_room_versions = config.get("allowed_room_versions", ["9", "10"])
+        if not _allowed_room_versions or not isinstance(_allowed_room_versions, list):
+            msg = "Allowed room versions must be formatted as a list. Please fix"
+            raise ConfigError(msg)
+
+        _config.allowed_room_versions.extend(
+            # Coercing into a string, in case the yaml loader thought it was an int
+            str(_room_ver)
+            for _room_ver in _allowed_room_versions
+        )
```

Comment:
```suggestion
        _config.allowed_room_versions = [
            # Coercing into a string, in case the yaml loader thought it was an int
            str(_room_ver)
            for _room_ver in _allowed_room_versions
        ]
```

Any reason you used `extend` there?

Reply:
I think I was setting the defaults a different way in a previous iteration. Since the empty list is already generated, this seemed more prudent than recreating the list. But I'm not married to it, this will be better

Reply:
fixed in [6948be3](https://github.com/famedly/synapse-invite-checker/pull/41/commits/6948be37766665fe0b984b07358fed77be35d375)